Gemfile.lock

```diff
@@ -30,10 +30,10 @@ GEM
       safe_yaml (~> 1.0.0)
     diff-lcs (1.3)
     docile (1.3.2)
-    google-protobuf (3.11.4)
+    google-protobuf (3.11.4-universal-darwin)
```

Comment:
I don't think we want to commit this native package (it's locked to a specific OS)